CSLabs.Api/Controllers/ModuleController.cs

```diff
@@ -121,5 +147,25 @@ public async Task<IActionResult> Upsert([FromBody] Module module)
             await DatabaseContext.Entry(module).Collection(m => m.Labs).LoadAsync();
             return Ok(module);
         }
+        
+        [HttpPost("{id}/tags")]
+        public async Task<IActionResult> AddTagToModule(int id, [FromBody] Tag tag)
```

Comment:
Depending on where the module is edited, this setting might get sent in the Upsert method since we try and make only one request to save an object.